typescript/ws-protocol/README.md

```diff
@@ -16,28 +14,13 @@ This package does not require a specific WebSocket server or client implementati
 $ npm install ws
 ```
 
-## Example servers
-
-### System monitor
-
-A system monitor script is provided as an illustrative example of a WebSocket server. To try it out:
-
-1. Run `npx @foxglove/ws-protocol-examples sysmon`.
-2. In a browser, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
-
-<img width="500" alt="Foxglove Studio displaying memory and CPU usage from the system monitor example" src="https://user-images.githubusercontent.com/14237/145313065-85c05645-6b29-4eb2-a498-849c83f8792d.png">
+## Example servers and client
```

Comment:
I know it's technically correct that there are 2 servers and 1 client, but it sounds weird to me to see one of these being plural and the other not 🤷